cmd/tekton-mcp-server/main.go

```diff
@@ -19,6 +21,12 @@ import (
 const ManagedByLabelKey = "app.kubernetes.io/managed-by"
 
 func main() {
+	var transport string
+	var sseAddr string
+	flag.StringVar(&transport, "transport", "stdio", "Transport type (stdio or sse)")
+	flag.StringVar(&sseAddr, "address", ":8080", "Address to bind the SSE server to")
```

Comment:
We may want to fail if the user specify `-address` without the `sse` transport.

Reply:
Added a condition to fail if it isn't set and removed the default